pkg/iac-providers/docker/v1/testdata/file-test-data/valid-Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM runatlantis/atlantis:v0.16.1
```

Comment:
Would it be possible to add `minSeverity`, `maxSeverity` and `skipRules` in one of the Dockerfiles for testing